docs/querying/functions.md

```diff
@@ -434,3 +434,15 @@ over time and return an instant vector with per-series aggregation results:
 
 Note that all values in the specified interval have the same weight in the
 aggregation even if the values are not equally spaced throughout the interval.
+
+## Trigonometric Functions
+
+- `acos(v instant-vector)`: calculates the arccosine, in radians, of all elements in `v` ([special cases](https://pkg.go.dev/math#Acos)).
+- `asin(v instant-vector)`: calculates the arcsine, in radians, of all elements in `v` ([special cases](https://pkg.go.dev/math#Asin)).
+- `atan(v instant-vector)`: calculates the arctangent, in radians, of all elements in `v` ([special cases](https://pkg.go.dev/math#Atan)).
+- `cos(v instant-vector)`: calculates the cosine, in radians, of all elements in `v` ([special cases](https://pkg.go.dev/math#Cos)).
```

Comment:
I would say that it's the elements in `v` that are in radians here, not the value of the calculation. The inverse functions calculate the value "in radians", but the forward functions calculate a value of "all the elements in `v` in radians".

Reply:
We can specify "in radians" as an introduction of the trig section.

Reply:
Yes that's probably simpler, avoids some otherwise awkward wording.

Reply:
```
All trigonometric operations are in radians.
```

WDYT? `rad()`, `deg()`, `pi()` aren't in radians so I don't think we can label all "functions" as in radians.

Reply:
How about:

"The trigonometric functions work in radians:
[docs for arccos, arcsin, arctan, cos, sin, tan]

The following are useful for converting between degrees and radians:
[deg, rad, pi]"

Reply:
That sounds great @leonerd